docs/sources/_index.md

```diff
@@ -15,10 +15,9 @@ cascade:
 
 # Grafana Agent
 
-Grafana Agent is a vendor-neutral, batteries-included telemetry collector with
-configuration inspired by [Terraform][]. It is designed to be flexible,
-performant, and compatible with multiple ecosystems such as Prometheus and
-OpenTelemetry.
+Grafana Agent is an OpenTelemetry Collector distribution with configuration
+inspired by [Terraform][]. It is designed to be flexible, performant, and
```

Comment:
Should we call out as primary goal brining Prometheus native pipeline to he OTel Collector in order to notable seamless integration between the Otel and Prometheus worlds?